kossy360/Questioner
# #163136994 integrate database and extra endpoints

#### What does this PR do?
 - Integrate a database for data persistence
 - Add extra endpoints
 - Secures all routes with JWT
#### Description of Task to be completed?
 - build a database with postgresql
 - have the following endpoints working
```
POST /api/v1/auth/signup
```
```
POST /api/v1/auth/login
```
```
POST /api/v1/comments
```
```
GET /api/v1/comments/<question-id>
```
```
POST /api/v1/notifications/<meetup-id>/register
```
```
GET /api/v1/notifications/<meetup-id>/get
```
```
PATCH /api/v1/meetups/<meetup-id>
```
```
PATCH /api/v1/users/update
```
```
PATCH /api/v1/notifications/<meetup-id>/reset
```
```
DELETE /api/v1/meetups/<meetup-id>
```
```
DELETE /api/v1/notifications/<meetup-id>/clear
```
- Secure all access routes with JWT
#### How should this be manually tested?
 - carry out all manual tests using POSTMAN, you should provide the JWT token in the 'auth' header of all requests
 - JWTs are sent as part of the response body after successful signup or login
 - All routes that create, modify or delete a meetup all require admin priviledges
 - to carry out automated tests using jasmine, you need to provide necessary database credentials in a .env file.
#### What are the relevant pivotal tracker stories?
#163136994
#163240264
#163240489
#163240749
#163241034
#163241248
#163241363
#163241514
#163243649
#163243692
#163243749
